src/GafferSceneUI/StandardLightVisualiser.cpp

```diff
@@ -337,6 +337,11 @@ IECoreGL::ConstRenderablePtr StandardLightVisualiser::visualise( const IECore::I
 	return result;
 }
 
+IECore::CompoundDataPtr StandardLightVisualiser::surfaceTexture( const std::string &type, const IECore::InternedString &attributeName, const IECoreScene::ShaderNetwork *shaderNetwork, const IECore::CompoundObject *attributes ) const
```

Comment:
@johnhaddon I'm not sure the signature for this is 100% yet, be good to get your thoughts.

Reply:
Improved in https://github.com/GafferHQ/gaffer/pull/3491/commits/8916d9093f2206136577a8b2bceb1a961d0973c6